docs/roman/references_general/dark_selection.inc

```diff
@@ -0,0 +1,13 @@
+.. _dark_selectors:
+
+Reference Selection Keyword Attributes for DARK
++++++++++++++++++++++++++++++++++++++++++++++++
+CRDS selects appropriate DARK references based on the following keyword attributes.
+DARK is not applicable for instruments not in the table.
+
+========== ===========================================
+Instrument  Keyword Attributes                                                                                           
+========== ===========================================
+WIFI	    instrument, detector, date, time
```

Comment:
wifi? :)

Reply:
Oh my gosh.. I laughed so hard at this comment.. when autocorrect attacks! Obviously I will change it to WFI.

Reply:
Autocorrect fail fixed. (And still makes me laugh.)